docs/api.md

```diff
@@ -4,16 +4,17 @@ Popover attaches to an element and displays on hover/blur.
 
 ## Attributes
 
-| Attribute  | Type      | Description                                      |
-|------------|-----------|--------------------------------------------------|
-| `addSpace` | `boolean` | If true, will add additional top and bottom space around the appearance of the popover in relation to the trigger.. |
+| Attribute     | Type      | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| `addSpace`    | `boolean` | If true, will add additional top and bottom space around the appearance of the popover in relation to the trigger |
+| `removeSpace` | `boolean` | If true, will remove top and bottom space around the appearance of the popover in relation to the trigger |
```

Comment:
We probably don't want anyone to ever add both of these attributes at the same time - which seems obvious - but someone will likely mess it up eventually. We be good to handle that scenario or at least add a `Do's and Don'ts` section to point out the futility of adding both of these.

Reply:
Yeah, I hear what you are saying. This gets to the point that I have been discussing with @geoffrich about using boolean attributes or string attributes. E.g. to address what you are saying the answer would be to update the API from `addSpace` and `removeSpace` to `space="add"` and `space="rmeove"`. 

That would require a breaking change in the API. Or we could defensively code against it. 

Either way, this is why we need to have a discussion about API development.